### DISCUSSION TITLE
Sonos Connect

### DISCUSSION THREAD

**Discussion by rehmaj** [music-assistant/support#2306] [Q&A] [answered]
I've searched and didn't see an answer. I have an Old Sonos Connect that I use as a Line In input to the Sonos ecosystem. I use it to "cast" my vinyl through the whole house those times when I don't want to listen to it on the 2 hardwired traditional speakers. 

How do I make this Connect as a provider?

This does work simply as expected in the Sonos app. I realize this is potentially an edge case or that I'm blind in my searching.

**Comment by OzGav** [comment, marked as answer]
Not currently possible. Current alternative is here https://music-assistant.io/faq/stream-to/#music-assistant

**Reply by rehmaj** [reply]
Sounds good. I'll stick with the Sonos app for the moment for those rare times I do it. Appreciate the answer.
